src/main/java/frc/robot/commands/CmdDriveToPosition.java

```diff
@@ -57,6 +58,8 @@ public void initialize()
         _xPID.setSetpoint(_targetPosition.getX(), fieldPosition.getX(), true);
         _yPID.setSetpoint(_targetPosition.getY(), fieldPosition.getY(), true);
         _rotatePID.setSetpoint(   _targetHeading,  _drive.getHeading(), true);
+
+        RobotLog.getInstance().log("Driving to " + _targetPosition + " with heading " + _targetHeading + " Currently at " +  Drive.getInstance().getFieldPosition() + " With heading " + Drive.getInstance().getHeading());
```

Comment:
We already have a member variable for the drive subsystem, so we can shorten this line a little bit. Additionally, let's format this string a little bit to remove some unnecessary decimal places and cut back on a few words:

```suggestion
        RobotLog.getInstance().log(String.format("Drive to position: Current position and heading: %s, %6.2f ; Target position and heading: %s, %6.2f", _drive.getFieldPosition().toString(), _drive.getHeading(), _targetPosition.toString(), _targetHeading));
```

If we do it this way, we'll end up with a message something like this:

```
Drive to position: Current position and heading: (236.34, 12.48), 138.42 ; Target position and heading: (200.00, 10.00), 120.00
```